custom_components/foxess/sensor.py

```diff
@@ -100,6 +101,31 @@
 
 token = None
 
+async def async_get_headers(url_path, token):
+    user_agent = user_agent_rotator.get_random_user_agent()
+    lang = "en"
+    path = url_path
+    path = path.replace("https://www.foxesscloud.com","")
+    timestamp = str(int(time.time()))
```

Comment:
Other implementations i've seen used 
```suggestion
    timestamp = str(int(time.time() * 1000))
```

Is this not needed at all?